template/package.json

```diff
@@ -1,20 +1 @@
-{
-  "private": true,
-  "name": "sanity-gatsby-blog",
-  "version": "1.0.17",
-  "scripts": {
-    "build": "lerna run build --parallel",
-    "dev": "lerna run dev --parallel",
-    "format": "lerna run format",
-    "build-studio": "lerna bootstrap && cd studio && npm run build",
-    "build-web": "lerna bootstrap && (cd studio && SANITY_AUTH_TOKEN=$SANITY_DEPLOY_STUDIO_TOKEN npm run graphql-deploy) && (cd web && npm run build)",
-    "graphql-deploy": "lerna run graphql-deploy",
-    "lint": "lerna run lint",
-    "postinstall": "lerna bootstrap",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "devDependencies": {
-    "@sanity/cli": "^2.0.0",
-    "lerna": "^3.13.1"
-  }
-}
+{"private":true,"name":"blog-with-gatsby","version":"1.0.17","scripts":{"build":"lerna run build --parallel","dev":"lerna run dev --parallel","format":"lerna run format","build-studio":"lerna bootstrap && cd studio && npm run build","build-web":"lerna bootstrap && (cd studio && SANITY_AUTH_TOKEN=$SANITY_DEPLOY_STUDIO_TOKEN npm run graphql-deploy) && (cd web && npm run build)","graphql-deploy":"lerna run graphql-deploy","lint":"lerna run lint","postinstall":"lerna bootstrap","test":"echo \"Error: no test specified\" && exit 1"},"devDependencies":{"@sanity/cli":"^2.0.0","lerna":"^3.13.1"}}
```

Comment:
Not sure why you removed the line spaces here?

Reply:
Stupid prettier :( I though I fixed it, but apparently I didn't. Would you suggest preserving the results of `npm run format`? It changes quite a bit of formatting :/